src/overlay006/ov6_02240C9C.c

```diff
@@ -902,27 +833,21 @@ static BOOL ov6_022418B4 (FieldSystem * fieldSystem, u32 param1)
 
     param1 = 8 - (param1);
 
-    if (fieldSystem->unk_78.unk_00 >= param1) {
-        return 1;
-    }
-
-    return 0;
+    return (fieldSystem->unk_78.unk_00 >= param1);
 }
 
-static BOOL ov6_022418DC (FieldSystem * fieldSystem, u32 param1)
+static BOOL ov6_022418DC (FieldSystem * fieldSystem, u32 encounterRate)
 {
-    if (inline_020564D0(100) >= param1) {
-        return 0;
+    if (inline_020564D0(100) >= encounterRate) {
+        return FALSE;
     } else {
-        return 1;
+        return TRUE;
     }
```

Comment:
**question:** Does this match if you return the inverted conditional? e.g. `return inline_020564D0(100) < encounterRate;`

Reply:
Unfortunately not. I checked different options but was not able to find something simpler that matches.

Reply:
Unfortunate. No foul play leaving it as-is.